From ff521728e6a8e7782515795c5392f14d3200f0a0 Mon Sep 17 00:00:00 2001
From: Amit Shah <amit.shah@redhat.com>
Date: Mon, 1 Aug 2011 11:40:53 -0300
Subject: [RHEL6 qemu-kvm PATCH 52/65] balloon: Fix header comment; add Copyright

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <8132aa40eeb76d86485328cd27ffa35fe542577d.1312198249.git.amit.shah@redhat.com>
Patchwork-id: 30726
O-Subject: [RHEL6.2 qemu PATCH 07/13] balloon: Fix header comment; add Copyright
Bugzilla: 694378
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

Signed-off-by: Amit Shah <amit.shah@redhat.com>
(cherry picked from commit e2b40e003aab79778d7f79fbf982afa1eb62cda9)

Bugzilla: 694378

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 balloon.c |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 balloon.c |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/balloon.c b/balloon.c
index e934878..2312584 100644
--- a/balloon.c
+++ b/balloon.c
@@ -1,7 +1,9 @@
 /*
- * QEMU System Emulator
+ * Generic Balloon handlers and management
  *
  * Copyright (c) 2003-2008 Fabrice Bellard
+ * Copyright (C) 2011 Red Hat, Inc.
+ * Copyright (C) 2011 Amit Shah <amit.shah@redhat.com>
  *
  * Permission is hereby granted, free of charge, to any person obtaining a copy
  * of this software and associated documentation files (the "Software"), to deal
@@ -31,7 +33,6 @@
 #include "balloon.h"
 #include "trace.h"
 
-
 static QEMUBalloonEvent *balloon_event_fn;
 static QEMUBalloonStatus *balloon_stat_fn;
 static void *balloon_opaque;
-- 
1.7.3.2

